src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random random = new Random();
```

Comment:
Avoid using static variables if they are not constants. Here, 'random' is a static variable but not a constant, and it's used to generate different values during the program execution. Consider making it a non-static field.